app/src/components/market/steps/items/funding_and_fee_step.tsx

```diff
@@ -61,7 +69,21 @@ const FundingAndFeeStep = (props: Props) => {
 
   const collateralBalance = useAsyncDerivedValue('', new BigNumber(0), calculateCollateralBalance)
 
-  const error = !spread || funding.isZero() || funding.gt(collateralBalance)
+  const isFundingGreaterThanBalance = funding.gt(collateralBalance)
+  const error = !spread || funding.isZero() || isFundingGreaterThanBalance
+
+  useEffect(() => {
+    let isSubscribed = true
+
+    const messageError = isFundingGreaterThanBalance
+      ? `You don't have enough collateral in your balance.`
+      : ''
+    if (isSubscribed) setFundingMessageError(messageError)
+
+    return () => {
+      isSubscribed = false
+    }
+  }, [isFundingGreaterThanBalance])
```

Comment:
This probably doesn't need a `useEffect`.

Reply:
Done